workflow-types/src/lib.rs

```diff
@@ -0,0 +1,100 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub enum Shell {
+    #[serde(rename(deserialize = "fish"))]
+    Fish,
+    #[serde(rename(deserialize = "bash"))]
+    Bash,
+    #[serde(rename(deserialize = "zsh"))]
+    Zsh,
+}
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct Workflow {
+    pub name: String,
+    pub command: String,
```

Comment:
this is the Rust struct equivalent of the YAML file, open to feedback if people think we should change it in any way.